packages/amplify-category-hosting/lib/S3AndCloudFront/configuration-manager.js

```diff
@@ -8,11 +8,68 @@ const configurables = {
 };
 
 async function init(context) {
-  const { projectConfig } = context.exeInfo;
-  const timeStamp = `-${moment().format('YYYYMMDDHHmmss')}-`;
-  let bucketName = `${projectConfig.projectName + timeStamp}hostingbucket`;
-  bucketName = bucketName.replace(/[^-a-z0-9]/g, '');
+  await setBucketName(context);
+  const configureModule = require(configurables.Website);
+  await configureModule.configure(context);
+}
+
+async function configure(context) {
+  await checkBucketName(context);
+  await configureHostingComponents(context);
+}
 
+async function checkBucketName(context) {
+  const { serviceMeta } = context.exeInfo;
+  const bucketCreated = serviceMeta && serviceMeta.output && serviceMeta.output.HostingBucketName;
+  if (!bucketCreated) {
+    const bucketName = context.exeInfo.template.Resources.S3Bucket.Properties.BucketName;
+    await setBucketName(context, bucketName);
+  }
+}
+
+async function configureHostingComponents(context, lastConfiguredSection) {
+  const options = Object.keys(configurables);
+  const done = "I'm done.";
```

Comment:
Can we change this text to something more meaningful?

Reply:
any suggestions?